### ISSUE TITLE
Missing resource in 1.4.0 jar

### ISSUE THREAD

**Issue by kwaxer** [docbook/xslTNG#101]
An attempt to run `java -jar ../libs/docbook-xslTNG-1.4.0.jar article.xml` with the ZIP distribution [1.4.0](https://github.com/docbook/xslTNG/releases/tag/1.4.0) results in
```
Exception in thread "main" java.lang.RuntimeException: Cannot read: jar:file:/.../libs/docbook-xslTNG-1.4.0.jar!/etc/uris.xml
        at org.docbook.xsltng.Main.getStream(Main.java:213)
        at org.docbook.xsltng.Main.createCatalog(Main.java:155)
        at org.docbook.xsltng.Main.<init>(Main.java:73)
        at org.docbook.xsltng.Main.main(Main.java:49)
```

(The previous version runs as expected with `java -jar ../libs/docbook-xslTNG-1.3.1.jar article.xml`.)

**Comment by ndw** [comment]
Yes, the 1.4.0 jar looks broken. I'm in the middle of some changes; I'll get 1.4.1 out asap.

**Comment by ndw** [comment]
Fixed in 1.4.1, I believe.
